### PR TITLE
Pet favorite page

### PR DESCRIPTION
**Pull request by mnca16**
### **In this PR:**

- Stores pet's card info on firestore: 

1. Name
2. Age
3. Breed
4. Distance 
5. Photos

- Creates favorites file.
- Reuses PetList component to display isFavoritedList state on favorites page.
- Fixes bug when the user refresh the pet's bio page.

### CODE REVIEW

File: src/components/NavItems.js [hunk 1]
```diff
@@ -12,6 +17,12 @@ export function NavItems() {
     { navLinks: "About", menuLinks: [] },
     { navLinks: "Login", menuLinks: [] },
   ]
+
+  //adds favorite to nav if user is logged in
+  if (isLoggedIn) {
+    navItems.push({ navLinks: "Favorites", menuLinks: [] })
+  }
```
Comment (mnca16) [inline src/components/NavItems.js:24]: Once we have the dropdown for the logout, profile, and favorite we can restructure this code.

**Review by tarikutk — APPROVED** [review]
Looks Good.

**Review by MarissaLZ — APPROVED** [review]
LGTM!

**Review by IreneS1 — APPROVED** [review]
Great job Monica!

**Pull request merged**